src/index.js

```diff
@@ -21,6 +22,7 @@ const store = createStore(
 ReactDOM.render(
   <Provider store={store}>
     <ThemeProvider theme={theme}>
+      <CssBaseline/>
```

Comment:
Could be typed <CssBaseline />. Now there is no space. Otherwise everything looks good.